src/DocumentFormat.OpenXml/Packaging/PresentationDocument.cs

```diff
@@ -309,58 +308,6 @@ public DigitalSignatureOriginPart AddDigitalSignatureOriginPart()
             return childPart;
         }
 
-        /// <summary>
-        /// Adds a ThumbnailPart to the PresentationDocument.
-        /// </summary>
-        /// <param name="contentType">The content type of the ThumbnailPart.</param>
-        /// <returns>The newly added ThumbnailPart.</returns>
-        public ThumbnailPart AddThumbnailPart(string contentType)
-        {
-            ThumbnailPart childPart = new ThumbnailPart();
-            InitPart(childPart, contentType);
-            return childPart;
-        }
-
-        /// <summary>
-        /// Adds a ThumbnailPart to the PresentationDocument.
-        /// </summary>
-        /// <param name="partType">The type of the ThumbnailPart.</param>
-        /// <returns>The newly added ThumbnailPart.</returns>
-        public ThumbnailPart AddThumbnailPart(ThumbnailPartType partType)
-        {
-            string contentType = ThumbnailPartTypeInfo.GetContentType(partType);
-            string partExtension = ThumbnailPartTypeInfo.GetTargetExtension(partType);
-            PartExtensions.Register(contentType, partExtension);
-
-            return AddThumbnailPart(contentType);
-        }
-
-        /// <summary>
-        /// Adds a new part of type <typeparamref name="T"/>.
-        /// </summary>
-        /// <typeparam name="T">The class of the part.</typeparam>
-        /// <param name="contentType">The content type of the part. Must match the defined content type if the part is fixed content type.</param>
-        /// <param name="id">The relationship id. The id will be automatically generated if this param is null.</param>
-        /// <returns>The added part.</returns>
-        /// <exception cref="OpenXmlPackageException">When the part is not allowed to be referenced by this part.</exception>
-        /// <exception cref="ArgumentOutOfRangeException">When the part is fixed content type and the passed in contentType does not match the defined content type.</exception>
-        /// <exception cref="ArgumentNullException">Thrown when "contentType" is null reference.</exception>
-        /// <remarks>Mainly used for adding not-fixed content type part - ImagePart, etc.</remarks>
-        public override T AddNewPart<T>(string contentType, string id)
```

Comment:
why'd you remove this? it does a check that ensures only the main part is the correct content type

Reply:
Doesn't look like you deleted the equivalent in the others, so this is probably a mistake